### DISCUSSION TITLE
Troubleshooting  with Postal SMTP Server

### DISCUSSION THREAD

**Discussion by jaydeepnp** [postalserver/postal#2948] [Help with using Postal]
Hi All,

I am using  postal server version  2.1.4. in  my production environment since more then 1 year now.

Few days back We  observed an issue with postal server,  We were seeing failed to connect error at client side when it was trying to connect.

Can someone please help where / what exactly I can  check on postal server for specific period of time if there was any issues with postal services.

I am  having  IP Pool Enabled with 2 IPs (2 Servers ) , Both the servers are behind AWS ELB, Clients connects to deliver mails. 

When we observed issue both the servers were not accepting connection intermittently which i want to investigate.

I am suspecting below.

Both servers are configured with RAW MESSAGE RETENTION SIZE=100GB , If row message size gets increased then configured value is it possible servers stops accepting connections? 

Please suggest help on this.

Regards,
Jaydeep Patel

**Comment by willpower232** [comment]
What client error are you seeing?

If storage was a problem, you would see that your servers hard drives are full or approaching it.

**Reply by jaydeepnp** [reply]
Failed to connect is error we get on client side.. java application is client submitting  mails to server where we see this error.

Server is having  storage available , my question here is if RAW MESSAGE RETENTION SIZE=100GB is configured and  due to increase of mail load if raw message size gets increased, will it clear old raw messages immediately to accept and store new mails  or there will be any issue due to it.?

I want troubleshoot issue i observed on 24th April 2024 on  my production  servers where client (java application) was not able to submit new mails and was getting error (Failed to connect ). I need help how i can  find  what it could have caused this issue.

What logs i can check .. where exactly i can check logs etc.. 

Also refer my earlier discussion #2530 in which i asked for errors observed in  smtp logs "Errno::EPIPE: Broken pipe" which probably means network issue and  unwanted connections to mails server.

Is there any way we can accept connections from specified clients only and all other connections are denied?

Regards,
Jaydeep
